test/modules/packages/Curl/setopt.chpl

```diff
@@ -0,0 +1,20 @@
+use Curl;
+
+proc main() {
+  const uniprotid = "P07204_TRBM_HUMAN";
+  const uniproturl = "http://www.uniprot.org/uniprot/" + uniprotid + ".fasta";
```

Comment:
Seems to me that we should probably create testing traffic against our own website